File: sdk/async-scala/src/main/scala/io/swagger/client/model/Shipment.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Shipment (
  ID: Option[String],
Shipper: Option[String],
DateShipped: Option[Date],
TrackingNumber: Option[String],
Cost: Option[Number],
Items: Option[List[ShipmentItem]],
xp: Option[Any]
)

File: sdk/slim/lib/Models/Shipment.php
<?php
/*
 * Shipment
 */
namespace \Models;

/*
 * Shipment
 */
class Shipment {
    /* @var string $iD  */
    private $iD;
/* @var string $shipper  */
    private $shipper;
/* @var \DateTime $dateShipped  */
    private $dateShipped;
/* @var string $trackingNumber  */
    private $trackingNumber;
/* @var Number $cost  */
    private $cost;
/* @var \\Models\ShipmentItem[] $items  */
    private $items;
/* @var object $xp  */
    private $xp;
}
